### DISCUSSION TITLE
Please publish vcpkg's Linux Arm64 executable on the release page?

### DISCUSSION THREAD

**Discussion by masx200** [microsoft/vcpkg#31105] [Ideas]
Please publish vcpkg's Linux Arm64 executable on the release page?

https://github.com/microsoft/vcpkg-tool/releases

**Comment by autoantwort** [comment]
They want to, but they have no idea how to build an arm version of vcpkg that is compatible with all the arm linux machines. So if you have instructions please submit them here or create a pr :)

**Reply by autoantwort** [reply]
See https://discord.com/channels/400588936151433218/687365466422902841/1078062757582209165 for reference

**Reply by autoantwort** [reply]
Or better here: https://discord.com/channels/400588936151433218/687365466422902841/1067595244729802822

**Comment by masx200** [comment]
https://github.com/microsoft/vcpkg/discussions/31064

I have successfully compiled vcpkg on my Snapdragon 865 android phone, using zerotermux and clang16.

**Reply by Osyotr** [reply]
That's cool but this binary will most likely not work on any other arm system.
